src/Swift.Bindings/tests/FrameworkTests/HikingApp/HikingApp.MacCatalyst/HikingApp.MacCatalyst.csproj

```diff
@@ -24,9 +24,14 @@
     </ImageAsset>
     <ImageAsset Include="Assets.xcassets\TrailImage.imageset\Contents.json">
       <Visible>false</Visible>
-    </ImageAsset>   
+    </ImageAsset>
   </ItemGroup>
 
+  <!-- Reference Swift.Bindings.MacOSX.Experimental from dotnet-experimental feed -->
```

Comment:
For tests, I think you want to reference local copy of the package that was just built.